### DISCUSSION TITLE
Unlink personal account from organization

### DISCUSSION THREAD

**Discussion by jeremywadsack** [community/community#54165] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

I started the process to enable Copilot for Business and it said it had to "link personal billing with the organization". I didn't realize this means my organization will now bill _my personal card_. I can't find any way to remove this link or change the billing information for the organization anymore. Anyone have any insights on how to fix this?

**Comment by jeremywadsack** [comment, marked as answer]
I got a response from GitHub support on this. In case anyone else runs into this it has something to do with accepting the Corporate TOS. Even though we had the organization billing separately for years.

> This is required due to your organization account being on our Standard Terms of Service, which means that only the members of the organization owners team (such as yourself) are considered organization owners, and as such one of you will be asked to share your own billing details. If this is intended as a company account, and you have separate billing details for your company, then you'll need to move the account to our Corporate Terms of Service. This will allow you to list the company itself as an owner of the account and share company-specific billing details.

> If you would like to make this change, you can follow the steps outlined in the article below to do so:

> [Upgrading to the Corporate Terms of Service](https://docs.github.com/en/organizations/managing-organization-settings/upgrading-to-the-corporate-terms-of-service)

**Comment by simonLeary42** [comment]
Leaving and re-joining the organization worked for me.
